force-app/main/default/classes/EDRD_myPatientsController.cls

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************************************
+* @Author:      Accenture 
+* @Date:        19/01/2024
+* @Description: The purpose of this class is to return list of patients to LWC edrd_myPatients
+* @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description] 
+***********************************************************************************************/
+public with sharing class EDRD_myPatientsController {
+    /**
+* @author: Accenture
+* @date: 19/01/2024
+* @description: The purpose of this method is to populate termination date based on Approval time/time unit
+* @Modification Log: [Date] - [Change Reference] - [Changed By] - [Description]
+*/  
+    @AuraEnabled(cacheable=true)
+    public static List<Contact> getPatientList() {
+        List<Contact> contactList = new List<Contact>();
+        List<user> users = [SELECT Id, AccountId FROM User WHERE Id = :UserInfo.getUserId() LIMIT 1];
```

Comment:
@rahulshyamrajsingh We need to validate CRUD permissions before SOQL/DML operations.  This is needed on lines 17 and 19.

Since Winter ‘23 (API Version 56) you can enforce user mode for database operations by using WITH USER_MODE in SOQL. This makes Apex to respect Field-level security (FLS) and object permissions of the running user. When using user mode, no violation is reported by this rule.